src/magnify/apps/core/models.py

```diff
@@ -76,6 +76,7 @@ class User(BaseModel, AbstractBaseUser, PermissionsMixin):
     language = models.CharField(
         max_length=10,
         choices=lazy(lambda: settings.LANGUAGES, tuple)(),
+        default=settings.LANGUAGE_CODE,
```

Comment:
If nothing is yet in production, maybe you could recreate the initial migration?

Reply:
True